app/controllers/web_git/commands_controller.rb

```diff
@@ -4,8 +4,15 @@ class CommandsController < ::ApplicationController
 
     def status
       Dir.chdir(Rails.root) do
+        
+        unless Dir.exist?(".git")
+          puts "Initializing a git repository"
+          `git init`
+          `git add -A`
+          puts "Make first commit"
+          `git commit -m "Initial commit"`
```

Comment:
How about an initial commit message of `Starting point` instead?

Reply:
Yes, I think that's more conceptually clear.